integration_tests/pages/page.ts

```diff
@@ -40,15 +40,15 @@ export default abstract class Page {
   getByName = (name: string): PageElement => cy.get(`[name=${name}]`)
 
   // Sub-navigation sections
-  selectSessionTemplatesTab = (): PageElement => cy.get('[data-test="tab-session-templates"]').click()
+  getSessionTemplatesTab = (): PageElement => cy.get('[data-test="tab-session-templates"]')
```

Comment:
Changing these so the `click()` is in the test script so they can be selected without necessarily clicking